src/ApiService/ApiService/Program.cs

```diff
@@ -6,13 +6,36 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using ApiService.OneFuzzLib;
-
-
+using Microsoft.Azure.Functions.Worker.Middleware;
+using Microsoft.Azure.Functions.Worker;
 
 namespace Microsoft.OneFuzz.Service;
 
 public class Program
 {
+    public class LoggingMiddleware : IFunctionsWorkerMiddleware
+    {
+        public async Async.Task Invoke(FunctionContext context, FunctionExecutionDelegate next)
+        {
+            //TODO
```

Comment:
Are there work items for the TODO's to make sure they don't get forgotten?

Reply:
for now just TODOs